packages/components/src/components/radio-group/radio-group.scss

```diff
@@ -3,6 +3,7 @@
     align-items: center;
     justify-content: flex-start;
     margin-top: 16px;
+    margin-bottom: 35px;
```

Comment:
This would break the design, since `radio-group` might be used on other places.

Reply:
Since the `margin-top` was used, I thought maybe it is ok to fix the margin in the css.